bridge/api/api.go

```diff
@@ -93,9 +93,10 @@ func (b *Api) handlePostMessage(c echo.Context) error {
 
 func (b *Api) handleMessages(c echo.Context) error {
 	b.Lock()
-	defer b.Unlock()
```

Comment:
why? what's wrong with the defer?

Reply:
defer is slower.  figure it is not needed since there are no early returns.

Reply:
Okay, but I don't really think speed matters here.
I prefer using locking and defer unlocking where possible (unless it gets locked too long) so that unlocks don't get forgotten and it's more readable.